src/main/java/com/faforever/client/teammatchmaking/TeamMatchmakingController.java

```diff
@@ -43,25 +58,68 @@
   @FXML
   public StackPane teamMatchmakingRoot;
   @FXML
-  public JFXListView<PartyMember> playerListView;
-  @FXML
   public JFXButton leavePartyButton;
   @FXML
   public JFXButton readyButton;
   @FXML
   public Label refreshingLabel;
-
+  public ToggleButton uefButton;
+  public ToggleButton cybranButton;
+  public ToggleButton aeonButton;
+  public ToggleButton seraphimButton;
+  @FXML
+  public ImageView avatarImageView;
+  @FXML
+  public ImageView countryImageView;
   @FXML
-  public JFXListView<MatchmakingQueue> queueListView;
+  public Label clanLabel;
+  @FXML
+  public Label usernameLabel;
+  @FXML
+  public Label teamRatingLabel;
+  @FXML
+  public Label gameCountLabel;
+  public Label ladderRatingLabel;
+  public HBox queueBox;
```

Comment:
With more than 2-4 queues this is gonna go wrong.

Reply:
There is space for six. Or do you mean something else?

Reply:
Is there space for six queues also on a width of 768?